ui/src/index.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import './index.css';
+import App from './App';
+import reportWebVitals from './reportWebVitals';
+
+ReactDOM.render(
+  <React.StrictMode>
+    <App />
```

Comment:
Interesting it is liike a xhtml tag macro devised by you

Reply:
Hahaha, it enforces JS strict mode, which is cool (almost) to have it enable...